Feature/verify address

ccali11 opened the pull request:
@shanejearley - Implemented the `verifyAddress` lambda function. Did not make any modifications to front-end or move the login logic out of the `ethers.ts` composable within the web app. Figured we could chat and decide how we want to move forward with that.

shanejearley reviewed (approved):
Wow, this worked out very nicely. 

For the comment you left about ethers hardhat, I think sometimes you need to either run an `npm i` (since I added a package) or restart the TS compiler in VSCode (since I added a hardhat.config.ts).

ccali11 commented:
> Wow, this worked out very nicely.
> 
> For the comment you left about ethers hardhat, I think sometimes you need to either run an `npm i` (since I added a package) or restart the TS compiler in VSCode (since I added a hardhat.config.ts).

@shanejearley - Restarting the TS Server in VSCode did the trick.
